webserver/webserver/src/test/java/io/helidon/webserver/TestHttpParseFineTuning.java

```diff
@@ -87,11 +87,11 @@ void testCustom() {
                 .validateHeaders(false)
                 .build();
 
-        testHeader(client, 8000, true);
+        testHeader(client, 16000, true);
         testInitialLine(client, 10, true);
 
-        testHeader(client, 8900, true);
-        testHeader(client, 8900, true);
+        testHeader(client, 17000, false);
+        testHeader(client, 17000, false);
```

Comment:
You are changing semantics of the test here, effectively removing original test case